adapters/httpapi_compact.c

```diff
@@ -976,11 +982,12 @@ static HTTPAPI_RESULT SendContentToXIO(HTTP_HANDLE_DATA* http_instance, const un
 }
 
 /*Codes_SRS_HTTPAPI_COMPACT_21_030: [ At the end of the transmission, the HTTPAPI_ExecuteRequest shall receive the response from the host. ]*/
-static HTTPAPI_RESULT ReceiveHeaderFromXIO(HTTP_HANDLE_DATA* http_instance, unsigned int* statusCode)
+static HTTPAPI_RESULT ReceiveHeaderFromXIO(HTTP_HANDLE_DATA* http_instance, unsigned int* statusCode, char* reasonPhrase)
```

Comment:
need add one more parameter telling length of the buffer, otherwise there could be buffer over run